tests/integration/test_keeper_back_to_back/test.py

```diff
@@ -90,6 +90,45 @@ def test_sequential_nodes(started_cluster):
         genuine_childs = list(sorted(genuine_zk.get_children("/test_sequential_nodes")))
         fake_childs = list(sorted(fake_zk.get_children("/test_sequential_nodes")))
         assert genuine_childs == fake_childs
+
+        genuine_zk.create("/test_sequential_nodes_1")
+        fake_zk.create("/test_sequential_nodes_1")
+
+        genuine_zk.create("/test_sequential_nodes_1/a", sequence=True)
+        fake_zk.create("/test_sequential_nodes_1/a", sequence=True)
+
+        genuine_zk.create("/test_sequential_nodes_1/a0000000002")
+        fake_zk.create("/test_sequential_nodes_1/a0000000002")
+
+        genuine_throw = False
+        fake_throw = False
+        try:
+            genuine_zk.create("/test_sequential_nodes_1/a", sequence=True)
+        except Exception as ex:
+            genuine_throw = True
+
+        try:
+            fake_zk.create("/test_sequential_nodes_1/a", sequence=True)
+        except Exception as ex:
+            fake_throw = True
+
+        assert genuine_throw == fake_throw
```

Comment:
It would be better to check that both `genuine_throw` and `fake_throw` are `True`

Reply:
Thanks.